infra/helm.tf

```diff
@@ -0,0 +1,52 @@
+provider "helm" {
+  kubernetes {
+    host = module.eks.cluster_endpoint
+    cluster_ca_certificate = base64decode(module.eks.cluster_certificate_authority_data)
+    exec {
+      api_version = "client.authentication.k8s.io/v1beta1"
+      args        = ["eks", "get-token", "--cluster-name", module.eks.cluster_name]
+      command     = "aws"
+    }
+  }
+}
+resource "helm_release" "alb_controler" {
+  name = "load-balancer-controller"
+  repository = "https://aws.github.io/eks-charts"
+  version = "1.7.2"
+  namespace = var.k8s_namespace
```

Comment:
This should not be used as variable as you will have two more helm chart and these will not change over time, just enter here what it's supposed to be.